hero_skills/src/hero_skills/hero.py

```diff
@@ -24,9 +27,13 @@ def __init__(self, wait_services=False):
 
         # add hero's arm
         hero_arm = arms.Arm(self.robot_name, self.tf_listener, self.get_joint_states, "left")
-        hero_arm.add_part('force_sensor', force_sensor.ForceSensor(self.robot_name, self.tf_listener, "/" + self.robot_name + "/wrist_wrench/raw"))
+        hero_arm.add_part('force_sensor', force_sensor.ForceSensor(
+            self.robot_name, self.tf_listener, "/" + self.robot_name + "/wrist_wrench/raw"
```

Comment:
```suggestion
            self.robot_name, self.tf_listener, self.robot_name + "/wrist_wrench/raw"
```
Frames should not start with leading slash

Reply:
Sharp, didn't introduce it but shouldn't be there. Fix in 7c24ed8e2f28d950b0b298e107bdd896c011e9b9